libs/i18n/assets/debug.json

```diff
@@ -41,43 +43,42 @@
   "build housing": "•build housing•",
   "Calculated by using FAO CLIMWAT 2.0 for Cropwat and Cropwat 8.0 and climate data for Nkhotakota Met Station.": "•Calculated by using FAO CLIMWAT 2.0 for Cropwat and Cropwat 8.0 and climate data for Nkhotakota Met Station.•",
   "Calculated with the date range of Start of rains and End of the rains": "•Calculated with the date range of Start of rains and End of the rains•",
-  "Calendar Lists": "•Calendar Lists•",
+  "Calendar name": "•Calendar name•",
   "Calendar Time": "•Calendar Time•",
-  "Calender name": "•Calender name•",
   "Cancel": "•Cancel•",
   "Capacity of 100mm on 1st March reduces 0.5 with evaporation rate of 5mm per day": "•Capacity of 100mm on 1st March reduces 0.5 with evaporation rate of 5mm per day•",
   "Cassava": "•Cassava•",
-  "Categories": "•Categories•",
   "cattle": "•cattle•",
   "Chance of receiving the water requirement in the days to maturity for this crop variety": "•Chance of receiving the water requirement in the days to maturity for this crop variety•",
   "Chart": "•Chart•",
   "chemicals": "•chemicals•",
   "chicken": "•chicken•",
+  "Clear": "•Clear•",
   "Click on the button below to retry": "•Click on the button below to retry•",
   "Climate": "•Climate•",
   "Close": "•Close•",
+  "Cloudy": "•Cloudy•",
   "Collection not found": "•Collection not found•",
   "Collections": "•Collections•",
   "Compare different options and plans": "•Compare different options and plans•",
   "Comparing different options and planning using Participatory Budgets": "•Comparing different options and planning using Participatory Budgets•",
   "compost manure making": "•compost manure making•",
+  "consumed": "•consumed•",
   "Consumed": "•Consumed•",
+  "Copy": "•Copy•",
   "Cost": "•Cost•",
   "Cotton": "•Cotton•",
   "Cowpeas": "•Cowpeas•",
   "cowpeas": "•cowpeas•",
-  "Create calender ": "•Create calender •",
```

Comment:
Thanks for catching this typo

Reply:
I noticed though that I can only add one activity for each crop in a given month, I was initially under the impression it could be more than one

Reply:
Good point, I think this is something I need to clarify with UoR myself as am a bit unsure too
Looking at the [manual page 14](https://picsa.app/manual?page=14) it looks like most have 1 activity drawn (although a couple have more), although the text says _main activity_ so a bit unclear.

Reply:
funnily enough we use the `cspell` package to automatically catch spelling mistakes although it missed this one - turns out _calender_ is still a valid word but is a type of machine they used in the 1600s to press cloth together 😅 
So I've added as a custom exception instead and should automatically detect in the future

Reply:
Thank you, that is an interesting fact